scripts/src/main/resources/scripts/command/dotnet

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+cd "${DEVON_IDE_HOME}" || exit 255
+
+
+function doSetup() {    
+  local dotnetCommand
+  local software_version="3.0.100"
+  local fileSetup
+  local download_url
+
+  dotnetCommand=$(dotnet --version)
+  
+  if [ "$dotnetCommand" != "$software_version" ]
+  then    
+    if doIsMacOs
+    then
+      fileSetup="dotnet-sdk-3.0.100-osx-x64.pkg"    
+      download_url="https://download.visualstudio.microsoft.com/download/pr/5c281f95-91c4-499d-baa2-31fec919047a/38c6964d72438ac30032bce516b655d9/dotnet-sdk-3.0.100-osx-x64.pkg"
```

Comment:
whoa. Great cryptic download URL that is provided here. For all other tools we support to configure the desired version via some variable. See #110 for details. For VSCode I had the same cryptic URLs at the beginning and found out that it is also possible to find download URLs with semantic version name instead that redirects to the cryptic download URL.
Is it possible to have multiple .NET installations on the same machine and prevent a "global system" installation?
We already do that for Java, Maven, Gradle, Eclipse, VSCode, IntelliJ, Node.js, etc.
If we do not follow this for .NET we will have a completely different UX for .NET and will cause lots of confusion (see the features that we promise for devonfw-ide and that you do not keep here).